.github/workflows/maturin.yml

```diff
@@ -0,0 +1,76 @@
+# This file is autogenerated by maturin v1.7.4 and manually modified by ia0.
+name: Maturin
+
+on:
+  pull_request:
```

Comment:
We need to decide on what exactly we want to trigger. Maybe something like:

```yaml
  push:
    branch:
      - main
    tags:
      - 'python-v*'
  workflow_dispatch:
```

Reply:
makes sense to me.